File: src/targets/csharp/httpclient/fixtures/nested.cs
using System.Net.Http.Headers;
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Get,
    RequestUri = new Uri("http://mockbin.com/har?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value"),
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}

File: src/targets/dart/http/fixtures/nested.dart
import 'package:http/http.dart' as http;

void main() async {

  final response = await http.get(
    Uri.parse('http://mockbin.com/har?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value'),
    headers: {},
  );

  print(response.statusCode);
  print(response.body);

}